.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Don't understand, why do we need to put this in `.tool`, but put `verify-gofmt` in `hack/`?
Why can't we all move them into `hack`? Is there any problem?